UI/CombatReport/GraphicalSummary.cpp

```diff
@@ -541,40 +541,43 @@ class OptionsBar : public GG::Wnd {
     OptionsBar(std::unique_ptr<BarSizer>& sizer) :
         GG::Wnd(),
         m_sizer(sizer)
-    {
-        m_toggles.push_back(new ToggleData(UserString("COMBAT_SUMMARY_PARTICIPANT_RELATIVE"),
-                                           UserString("COMBAT_SUMMARY_PARTICIPANT_EQUAL"),
-                                           UserString("COMBAT_SUMMARY_PARTICIPANT_RELATIVE_TIP"),
-                                           UserString("COMBAT_SUMMARY_PARTICIPANT_EQUAL_TIP"),
-                                           TOGGLE_BAR_WIDTH_PROPORTIONAL, &m_sizer, this));
-        m_toggles.push_back(new ToggleData(UserString("COMBAT_SUMMARY_HEALTH_SMOOTH"),
-                                           UserString("COMBAT_SUMMARY_HEALTH_BAR"),
-                                           UserString("COMBAT_SUMMARY_HEALTH_SMOOTH_TIP"),
-                                           UserString("COMBAT_SUMMARY_HEALTH_BAR_TIP"),
-                                           TOGGLE_BAR_HEALTH_SMOOTH, &m_sizer, this));
-        m_toggles.push_back(new ToggleData(UserString("COMBAT_SUMMARY_BAR_HEIGHT_PROPORTIONAL"),
-                                           UserString("COMBAT_SUMMARY_BAR_HEIGHT_EQUAL"),
-                                           UserString("COMBAT_SUMMARY_BAR_HEIGHT_PROPORTIONAL_TIP"),
-                                           UserString("COMBAT_SUMMARY_BAR_HEIGHT_EQUAL_TIP"),
-                                           TOGGLE_BAR_HEIGHT_PROPORTIONAL, &m_sizer, this));
-        m_toggles.push_back(new ToggleData(UserString("COMBAT_SUMMARY_GRAPH_HEIGHT_PROPORTIONAL"),
-                                           UserString("COMBAT_SUMMARY_GRAPH_HEIGHT_EQUAL"),
-                                           UserString("COMBAT_SUMMARY_GRAPH_HEIGHT_PROPORTIONAL_TIP"),
-                                           UserString("COMBAT_SUMMARY_GRAPH_HEIGHT_EQUAL_TIP"),
-                                           TOGGLE_GRAPH_HEIGHT_PROPORTIONAL, &m_sizer, this));
+    {}
+
+    void CompleteConstruction() override {
+        m_toggles.push_back(std::make_shared<ToggleData>(UserString("COMBAT_SUMMARY_PARTICIPANT_RELATIVE"),
```

Comment:
seems like a good case for moving the first parameter to the next line and indenting 4 spaces instead of all the way to after the `(`